docs/components/bn-file-input.md

```diff
@@ -0,0 +1,326 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import BnFileInput from '../../src/components/BnFileInput/BnFileInput.vue'
+import { Form, ErrorMessage } from 'vee-validate';
+
+const file = ref(undefined);
+const fileList = ref(undefined);
+const validate = ref(undefined);
+
+const warningIcon = 'M11.9998 9.00006V12.7501M2.69653 16.1257C1.83114 17.6257 2.91371 19.5001 4.64544 19.5001H19.3541C21.0858 19.5001 22.1684 17.6257 21.303 16.1257L13.9487 3.37819C13.0828 1.87736 10.9167 1.87736 10.0509 3.37819L2.69653 16.1257ZM11.9998 15.7501H12.0073V15.7576H11.9998V15.7501Z';
+
+function isRequired(val: File[] | File | undefined) {
+  if (val === undefined || val.length === 0) {
```

Comment:
con esta regla no tiraría error cuando es single file y si hay un file seleccionado? por undefined "length" for File object, algo así

Reply:
No, because object.length is undefined. So `undefined || true` = `true`. Maybe It is not clear enough. Do you think that would be better if I change it?. That's for internal use, so it's okay with me.

Reply:
oh you're right, it's fine as is then!